docker/ci

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+# See https://docs.codecov.io/docs/testing-with-docker, also note that it is
+# critical to set CI=true because codecov-ruby won't detect GitHub Actions
+# without it per:
+# https://github.com/codecov/codecov-ruby/blob/484767f1c3d7992a9d7fedd6dc72d35a80d04f70/lib/codecov.rb#L68-L69
+
+docker run $CI_ENV \
+           -e CI=true \
+           -v $DIR/..:/code \
+           -w /code \
+           openstax/recipes.ci:latest \
+           /bin/bash -c "exec rspec"
```

Comment:
what is `exec` here?  Might have expected `bundle exec rspec` or just `rspec`

Reply:
not sure haha, I'll take it out